docs/en/interfaces/mysql.md

```diff
@@ -35,6 +35,8 @@ Alternatively, in order to enable the MySQL interface for an existing service:
 3. After entering the password, you will get prompted the MySQL connection string for this service
 ![Connection screen -  MySQL Enabled](./images/mysql5.png)
 
+If the connection is asking for a server certificate, ClickHouse Cloud's SSL certificate is signed by [LetsEncrypt](https://letsencrypt.org/certificates/). You can download this root cert [here](https://letsencrypt.org/certs/isrgrootx1.pem).
```

Comment:
```suggestion
If the connection is asking for a server certificate, ClickHouse Cloud's SSL certificate is signed by [LetsEncrypt](https://letsencrypt.org/certificates/).
```

I'd recommend not directly linking to a cert download to mitigate man-in-the-middle attacks (and sketchy stuff).